src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -153,4 +155,12 @@ public static void showPatientAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredPatientList().size());
     }
 
+    public static void showPatientAtIC(Model model, IcNumber icNumber) {
+        List<Patient> lastShownList = model.getFilteredPatientList();
+        Patient patient = model.getPatient(icNumber, lastShownList);
+        final String[] splitName = patient.getName().fullName.split("\\s+");
+        model.updateFilteredPatientList(new NameContainsKeywordsPredicate(Arrays.asList(splitName[0])));
+        assertEquals(1, model.getFilteredPatientList().size());
+    }
+
```

Comment:
I like how you tried to implement a testing utility to show the patient of a IC. But considering how you are using the filtered patient list, would it be good if we can use the full list instead? since filtered patient list does not contain all patients